packages/vscode-extension/src/debug/commonUtils.ts

```diff
@@ -282,9 +282,31 @@ function getSettingWithUserData(jsonSelector: (jsonObject: any) => any): string
   return undefined;
 }
 
+// This is for the new folder structure for multi-env
+function getLocalSetting(jsonSelector: (jsonObject: any) => any): string | undefined {
```

Comment:
CLI may have the same issue.

Reply:
CLI will be fixed later. Currently we focus on toolkit